Sources/Instrumentation/InstrumentationSystem.swift

```diff
@@ -34,27 +35,37 @@ public enum InstrumentationSystem {
                 you need to use multiple instruments.
                 """
             )
+            if let tracer = instrument as? TracingInstrument {
+                self._tracer = tracer
+            }
             self._instrument = instrument
+
             self.isInitialized = true
         }
     }
 
+    // for our testing we want to allow multiple bootstrapping
+    internal static func bootstrapInternal(_ instrument: Instrument) {
```

Comment:
we need this in order to test instrumented things.

The global one is "just set once forever" after all